types/socket.io/index.d.ts

```diff
@@ -81,7 +81,7 @@ interface SocketIOStatic {
 
 declare namespace SocketIO {
     interface Server {
-        engine: { ws: any };
+        engine: { ws: any, generateId: () => {} };
```

Comment:
this is not correct.
Based on the docs which you have provided the link to: 

1. takes the request as param
2. has to return a string
![image](https://user-images.githubusercontent.com/9385106/89815734-8fcca000-db45-11ea-9855-cd9b12a008ce.png)

Reply:
My apologies you are right. I changed the code to be more compliant with the docs.